From 3965839d3930caf2b513ec044fd415db10188bf6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lu=C3=ADs=20Pereira?= <luis.artur.pereira@gmail.com>
Date: Thu, 1 Dec 2016 00:58:51 +0000
Subject: [PATCH 2/4] Fixes FTBFS when libmenu-cache was installed to
 non-standard location

libmenu-cache pkg-config file defines Cflags as:
    Cflags: -I${includedir}/menu-cache
So #include <menu-cache/menu-cache.h> works only if ${includedir} is
included by default, e.g. standard locations. When libmenu-cache was
installed to a non-standard location ${includedir} is not part of the
default include path and #include <menu-cache/menu-cache.h> fails.
---
 providers.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/providers.h b/providers.h
index d693c0e..2234cfe 100644
--- a/providers.h
+++ b/providers.h
@@ -37,7 +37,7 @@
 #include <QIcon>
 
 #ifdef HAVE_MENU_CACHE
-#include <menu-cache/menu-cache.h>
+#include <menu-cache.h>
 #endif
 
 #define MAX_RANK 0xFFFF
-- 
2.8.3

